subsys/disk/disk_access_stm32_sdmmc.c

```diff
@@ -34,6 +35,10 @@ struct stm32_sdmmc_priv {
 		int flags;
 	} pe;
 	struct stm32_pclken pclken;
+	struct {
+		const struct soc_gpio_pinctrl *list;
+		size_t len;
+	} pinctrl;
```

Comment:
I like the idea of the struct for pinctrl, but would make it common and then provide an initializer macro so that all drivers use the same.

Reply:
Sure could be done once all are in.